teamcity: failed tests on master: testrace/TestRemovePlaceholderRace

The following tests appear to have failed:

[#360669](https://teamcity.cockroachdb.com/viewLog.html?buildId=360669):

```
--- FAIL: testrace/TestRemovePlaceholderRace (0.000s)
Race detected!

------- Stdout: -------
I170925 20:35:43.676218 60940 storage/store.go:1209  [s1] [n1,s1]: failed initial metrics computation: [n1,s1]: system config not yet available
I170925 20:35:43.676707 60940 gossip/gossip.go:331  [n1] NodeDescriptor set to node_id:1 address:<network_field:"tcp" address_field:"127.0.0.1:54488" > attrs:<> locality:<> ServerVersion:<major_val:0 minor_val:0 patch:0 unstable:0 > 
W170925 20:35:43.720525 60940 gossip/gossip.go:1277  [n2] no incoming or outgoing connections
I170925 20:35:43.722646 61095 gossip/client.go:129  [n2] started gossip client to 127.0.0.1:54488
I170925 20:35:43.751744 60940 gossip/gossip.go:331  [n2] NodeDescriptor set to node_id:2 address:<network_field:"tcp" address_field:"127.0.0.1:57865" > attrs:<> locality:<> ServerVersion:<major_val:0 minor_val:0 patch:0 unstable:0 > 
W170925 20:35:43.804074 60940 gossip/gossip.go:1277  [n3] no incoming or outgoing connections
I170925 20:35:43.806260 61052 gossip/client.go:129  [n3] started gossip client to 127.0.0.1:54488
I170925 20:35:43.825756 60940 gossip/gossip.go:331  [n3] NodeDescriptor set to node_id:3 address:<network_field:"tcp" address_field:"127.0.0.1:51283" > attrs:<> locality:<> ServerVersion:<major_val:0 minor_val:0 patch:0 unstable:0 > 
I170925 20:35:43.885101 61373 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 608cc45c at index 15
I170925 20:35:43.888488 61373 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 34, log entries: 5, rate-limit: 2.0 MiB/sec, 2ms
I170925 20:35:43.889643 61422 storage/replica_raftstorage.go:729  [s2,r1/?:{-}] applying preemptive snapshot at index 15 (id=608cc45c, encoded size=5681, 1 rocksdb batches, 5 log entries)
I170925 20:35:43.891153 61422 storage/replica_raftstorage.go:735  [s2,r1/?:/M{in-ax}] applied preemptive snapshot in 1ms [clear=0ms batch=0ms entries=0ms commit=0ms]
I170925 20:35:43.895377 61373 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):2): read existing descriptor r1:/M{in-ax} [(n1,s1):1, next=2]
I170925 20:35:43.921026 61310 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):2): updated=[(n1,s1):1 (n2,s2):2] next=3
I170925 20:35:43.939440 61445 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 255877bc at index 17
I170925 20:35:43.943060 61445 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n3,s3):?: kv pairs: 37, log entries: 7, rate-limit: 2.0 MiB/sec, 3ms
I170925 20:35:43.944644 61278 storage/replica_raftstorage.go:729  [s3,r1/?:{-}] applying preemptive snapshot at index 17 (id=255877bc, encoded size=7228, 1 rocksdb batches, 7 log entries)
I170925 20:35:43.946473 61278 storage/replica_raftstorage.go:735  [s3,r1/?:/M{in-ax}] applied preemptive snapshot in 2ms [clear=0ms batch=0ms entries=1ms commit=0ms]
I170925 20:35:43.949802 61445 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n3,s3):3): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n2,s2):2, next=3]
I170925 20:35:43.956340 61460 storage/raft_transport.go:455  raft transport stream to node 1 established
I170925 20:35:43.992745 61464 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n3,s3):3): updated=[(n1,s1):1 (n2,s2):2 (n3,s3):3] next=4
I170925 20:35:44.153687 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):2): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n2,s2):2, (n3,s3):3, next=4]
I170925 20:35:44.182774 61453 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):2): updated=[(n1,s1):1 (n3,s3):3] next=4
I170925 20:35:44.202685 61462 storage/store.go:3237  [s2,r1/2:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:44.204006 61298 storage/store.go:2168  [replicaGC,s2,r1/2:/M{in-ax}] removing replica
I170925 20:35:44.205042 61298 storage/replica.go:762  [replicaGC,s2,r1/2:/M{in-ax}] removed 34 (26+8) keys in 0ms [clear=0ms commit=0ms]
I170925 20:35:44.206515 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 12ba041e at index 22
I170925 20:35:44.210639 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 1d27a3f9 at index 22
I170925 20:35:44.224042 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 44, log entries: 12, rate-limit: 2.0 MiB/sec, 12ms
I170925 20:35:44.225300 61506 storage/replica_raftstorage.go:729  [s2,r1/?:{-}] applying preemptive snapshot at index 22 (id=1d27a3f9, encoded size=10900, 1 rocksdb batches, 12 log entries)
I170925 20:35:44.227397 61506 storage/replica_raftstorage.go:735  [s2,r1/?:/M{in-ax}] applied preemptive snapshot in 2ms [clear=0ms batch=0ms entries=1ms commit=0ms]
I170925 20:35:44.231639 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):4): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=4]
I170925 20:35:44.244704 61456 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):4): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):4] next=5
I170925 20:35:44.279657 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):4): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):4, next=5]
I170925 20:35:44.325079 61432 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):4): updated=[(n1,s1):1 (n3,s3):3] next=5
I170925 20:35:44.346805 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 09f7e0b7 at index 28
I170925 20:35:44.348649 61298 storage/store.go:2168  [replicaGC,s2,r1/4:/M{in-ax}] removing replica
I170925 20:35:44.349612 61298 storage/replica.go:762  [replicaGC,s2,r1/4:/M{in-ax}] removed 34 (26+8) keys in 0ms [clear=0ms commit=0ms]
I170925 20:35:44.372265 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 52, log entries: 18, rate-limit: 2.0 MiB/sec, 25ms
I170925 20:35:44.375869 61529 storage/replica_raftstorage.go:729  [s2,r1/?:{-}] applying preemptive snapshot at index 28 (id=09f7e0b7, encoded size=15060, 1 rocksdb batches, 18 log entries)
I170925 20:35:44.381776 61529 storage/replica_raftstorage.go:735  [s2,r1/?:/M{in-ax}] applied preemptive snapshot in 6ms [clear=0ms batch=0ms entries=5ms commit=0ms]
I170925 20:35:44.386501 61462 storage/store.go:3237  [s2,r1/?:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:44.390533 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):5): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=5]
I170925 20:35:44.418744 61298 storage/store.go:2168  [replicaGC,s2,r1/?:/M{in-ax}] removing replica
I170925 20:35:44.428353 61298 storage/replica.go:762  [replicaGC,s2,r1/?:/M{in-ax}] removed 34 (26+8) keys in 9ms [clear=0ms commit=9ms]
I170925 20:35:44.430075 61502 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):5): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):5] next=6
I170925 20:35:44.452452 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):5): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):5, next=6]
I170925 20:35:44.481009 61087 storage/replica_raftstorage.go:523  [raftsnapshot,s1,r1/1:/M{in-ax}] generated Raft snapshot 23aface2 at index 31
I170925 20:35:44.482548 61195 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):5): updated=[(n1,s1):1 (n3,s3):3] next=6
I170925 20:35:44.487387 61087 storage/store.go:3503  [raftsnapshot,s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):5: kv pairs: 58, log entries: 21, rate-limit: 8.0 MiB/sec, 5ms
I170925 20:35:44.490231 61197 storage/replica_raftstorage.go:729  [s2,r1/5:{-}] applying Raft snapshot at index 31 (id=23aface2, encoded size=17626, 1 rocksdb batches, 21 log entries)
I170925 20:35:44.507964 61197 storage/replica_raftstorage.go:735  [s2,r1/5:/M{in-ax}] applied Raft snapshot in 18ms [clear=0ms batch=0ms entries=16ms commit=0ms]
I170925 20:35:44.510360 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot be642ffb at index 32
I170925 20:35:44.512275 61462 storage/store.go:3237  [s2,r1/5:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:44.520252 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 58, log entries: 22, rate-limit: 2.0 MiB/sec, 7ms
I170925 20:35:44.520973 61550 storage/replica_raftstorage.go:729  [s2,r1/5:/M{in-ax}] applying preemptive snapshot at index 32 (id=be642ffb, encoded size=18244, 1 rocksdb batches, 22 log entries)
I170925 20:35:44.524173 61550 storage/replica_raftstorage.go:735  [s2,r1/5:/M{in-ax}] applied preemptive snapshot in 3ms [clear=0ms batch=0ms entries=2ms commit=0ms]
I170925 20:35:44.524978 61298 storage/store.go:2168  [replicaGC,s2,r1/5:/M{in-ax}] removing replica
I170925 20:35:44.525856 61298 storage/replica.go:762  [replicaGC,s2,r1/5:/M{in-ax}] removed 34 (26+8) keys in 0ms [clear=0ms commit=0ms]
I170925 20:35:44.533670 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):6): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=6]
W170925 20:35:44.538028 61459 storage/store.go:3244  [s1] got error from r1, replica (n2,s2):5: raft group deleted
I170925 20:35:44.543546 61618 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):6): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):6] next=7
I170925 20:35:44.562279 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):6): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):6, next=7]
I170925 20:35:44.574048 61087 storage/replica_raftstorage.go:523  [raftsnapshot,s1,r1/1:/M{in-ax}] generated Raft snapshot a879a73c at index 34
I170925 20:35:44.607027 61636 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):6): updated=[(n1,s1):1 (n3,s3):3] next=7
I170925 20:35:44.633424 61087 storage/store.go:3503  [raftsnapshot,s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):6: kv pairs: 61, log entries: 24, rate-limit: 8.0 MiB/sec, 3ms
I170925 20:35:44.642242 61589 storage/replica_raftstorage.go:729  [s2,r1/6:{-}] applying Raft snapshot at index 34 (id=a879a73c, encoded size=19868, 1 rocksdb batches, 24 log entries)
I170925 20:35:44.645665 61589 storage/replica_raftstorage.go:735  [s2,r1/6:/M{in-ax}] applied Raft snapshot in 3ms [clear=0ms batch=0ms entries=2ms commit=0ms]
I170925 20:35:44.660041 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot cbb2711c at index 37
I170925 20:35:44.662988 61462 storage/store.go:3237  [s2,r1/6:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:44.666054 61298 storage/store.go:2168  [replicaGC,s2,r1/6:/M{in-ax}] removing replica
I170925 20:35:44.667136 61298 storage/replica.go:762  [replicaGC,s2,r1/6:/M{in-ax}] removed 34 (26+8) keys in 1ms [clear=0ms commit=0ms]
I170925 20:35:44.668802 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 3344715c at index 37
I170925 20:35:44.672834 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 65, log entries: 27, rate-limit: 2.0 MiB/sec, 3ms
I170925 20:35:44.675034 61625 storage/replica_raftstorage.go:729  [s2,r1/?:{-}] applying preemptive snapshot at index 37 (id=3344715c, encoded size=21916, 1 rocksdb batches, 27 log entries)
I170925 20:35:44.678599 61625 storage/replica_raftstorage.go:735  [s2,r1/?:/M{in-ax}] applied preemptive snapshot in 3ms [clear=0ms batch=0ms entries=2ms commit=0ms]
I170925 20:35:44.682850 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):7): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=7]
I170925 20:35:44.723745 61610 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):7): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):7] next=8
I170925 20:35:44.769445 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):7): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):7, next=8]
I170925 20:35:44.779880 61669 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):7): updated=[(n1,s1):1 (n3,s3):3] next=8
I170925 20:35:44.791291 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot aa0de5af at index 43
I170925 20:35:44.797508 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 73, log entries: 33, rate-limit: 2.0 MiB/sec, 5ms
I170925 20:35:44.802613 61298 storage/store.go:2168  [replicaGC,s2,r1/7:/M{in-ax}] removing replica
I170925 20:35:44.803759 61298 storage/replica.go:762  [replicaGC,s2,r1/7:/M{in-ax}] removed 34 (26+8) keys in 1ms [clear=0ms commit=0ms]
I170925 20:35:44.806381 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):8): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=8]
I170925 20:35:44.816490 61673 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):8): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):8] next=9
W170925 20:35:44.834916 61459 storage/store.go:3244  [s1] got error from r1, replica (n2,s2):7: raft group deleted
I170925 20:35:44.844423 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):8): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):8, next=9]
I170925 20:35:44.856611 61087 storage/replica_raftstorage.go:523  [raftsnapshot,s1,r1/1:/M{in-ax}] generated Raft snapshot 7f8d597d at index 45
I170925 20:35:44.866269 61703 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):8): updated=[(n1,s1):1 (n3,s3):3] next=9
I170925 20:35:44.886222 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 2ee32034 at index 47
I170925 20:35:44.890066 61087 storage/store.go:3503  [raftsnapshot,s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):8: kv pairs: 76, log entries: 35, rate-limit: 8.0 MiB/sec, 8ms
I170925 20:35:44.891416 61629 storage/replica_raftstorage.go:729  [s2,r1/8:{-}] applying Raft snapshot at index 45 (id=7f8d597d, encoded size=27700, 1 rocksdb batches, 35 log entries)
I170925 20:35:44.895605 61629 storage/replica_raftstorage.go:735  [s2,r1/8:/M{in-ax}] applied Raft snapshot in 4ms [clear=0ms batch=0ms entries=3ms commit=0ms]
I170925 20:35:44.898005 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 575875ea at index 47
I170925 20:35:44.902176 61462 storage/store.go:3237  [s2,r1/8:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:44.905130 61298 storage/store.go:2168  [replicaGC,s2,r1/8:/M{in-ax}] removing replica
I170925 20:35:44.906218 61298 storage/replica.go:762  [replicaGC,s2,r1/8:/M{in-ax}] removed 34 (26+8) keys in 1ms [clear=0ms commit=0ms]
I170925 20:35:44.912516 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 79, log entries: 37, rate-limit: 2.0 MiB/sec, 14ms
I170925 20:35:44.914312 61631 storage/replica_raftstorage.go:729  [s2,r1/?:{-}] applying preemptive snapshot at index 47 (id=575875ea, encoded size=29260, 1 rocksdb batches, 37 log entries)
I170925 20:35:44.918245 61631 storage/replica_raftstorage.go:735  [s2,r1/?:/M{in-ax}] applied preemptive snapshot in 4ms [clear=0ms batch=0ms entries=3ms commit=0ms]
I170925 20:35:44.922841 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):9): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=9]
I170925 20:35:44.956415 61633 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):9): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):9] next=10
I170925 20:35:44.970698 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):9): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):9, next=10]
I170925 20:35:44.998148 61692 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):9): updated=[(n1,s1):1 (n3,s3):3] next=10
I170925 20:35:45.011683 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 81d8bb89 at index 52
I170925 20:35:45.018462 61462 storage/store.go:3237  [s2,r1/9:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:45.022421 61298 storage/store.go:2168  [replicaGC,s2,r1/9:/M{in-ax}] removing replica
I170925 20:35:45.023660 61298 storage/replica.go:762  [replicaGC,s2,r1/9:/M{in-ax}] removed 34 (26+8) keys in 1ms [clear=0ms commit=0ms]
I170925 20:35:45.028344 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 85, log entries: 42, rate-limit: 2.0 MiB/sec, 10ms
I170925 20:35:45.030818 61696 storage/replica_raftstorage.go:729  [s2,r1/?:{-}] applying preemptive snapshot at index 52 (id=81d8bb89, encoded size=32461, 1 rocksdb batches, 42 log entries)
I170925 20:35:45.045895 61696 storage/replica_raftstorage.go:735  [s2,r1/?:/M{in-ax}] applied preemptive snapshot in 15ms [clear=0ms batch=0ms entries=3ms commit=11ms]
I170925 20:35:45.059781 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):10): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=10]
I170925 20:35:45.079871 61735 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):10): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):10] next=11
I170925 20:35:45.092901 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):10): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):10, next=11]
I170925 20:35:45.117086 61765 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):10): updated=[(n1,s1):1 (n3,s3):3] next=11
I170925 20:35:45.126907 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot bf828e7f at index 57
I170925 20:35:45.138145 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 92, log entries: 47, rate-limit: 2.0 MiB/sec, 5ms
I170925 20:35:45.139202 61462 storage/store.go:3237  [s2,r1/10:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:45.139530 61298 storage/store.go:2168  [replicaGC,s2,r1/10:/M{in-ax}] removing replica
I170925 20:35:45.140730 61298 storage/replica.go:762  [replicaGC,s2,r1/10:/M{in-ax}] removed 34 (26+8) keys in 1ms [clear=0ms commit=0ms]
I170925 20:35:45.142625 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):11): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=11]
I170925 20:35:45.164742 61794 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):11): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):11] next=12
I170925 20:35:45.184616 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):11): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):11, next=12]
I170925 20:35:45.193204 61087 storage/replica_raftstorage.go:523  [raftsnapshot,s1,r1/1:/M{in-ax}] generated Raft snapshot 6a232051 at index 61
I170925 20:35:45.202367 61087 storage/store.go:3503  [raftsnapshot,s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):11: kv pairs: 99, log entries: 51, rate-limit: 8.0 MiB/sec, 8ms
I170925 20:35:45.203261 61796 storage/replica_raftstorage.go:729  [s2,r1/11:{-}] applying Raft snapshot at index 61 (id=6a232051, encoded size=39187, 1 rocksdb batches, 51 log entries)
I170925 20:35:45.208544 61796 storage/replica_raftstorage.go:735  [s2,r1/11:/M{in-ax}] applied Raft snapshot in 5ms [clear=0ms batch=0ms entries=4ms commit=0ms]
I170925 20:35:45.221390 61769 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):11): updated=[(n1,s1):1 (n3,s3):3] next=12
I170925 20:35:45.250296 61462 storage/store.go:3237  [s2,r1/11:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:45.255235 61298 storage/store.go:2168  [replicaGC,s2,r1/11:/M{in-ax}] removing replica
I170925 20:35:45.256512 61298 storage/replica.go:762  [replicaGC,s2,r1/11:/M{in-ax}] removed 35 (26+9) keys in 1ms [clear=0ms commit=0ms]
I170925 20:35:45.256705 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 57db5f96 at index 62
I170925 20:35:45.273544 61760 storage/replica_raftstorage.go:729  [s2,r1/?:{-}] applying preemptive snapshot at index 62 (id=57db5f96, encoded size=39805, 1 rocksdb batches, 52 log entries)
I170925 20:35:45.273995 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 99, log entries: 52, rate-limit: 2.0 MiB/sec, 13ms
I170925 20:35:45.296769 61760 storage/replica_raftstorage.go:735  [s2,r1/?:/M{in-ax}] applied preemptive snapshot in 23ms [clear=0ms batch=0ms entries=21ms commit=1ms]
I170925 20:35:45.310870 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):12): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=12]
I170925 20:35:45.321397 61815 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):12): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):12] next=13
I170925 20:35:45.338363 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):12): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):12, next=13]
I170925 20:35:45.357193 61801 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):12): updated=[(n1,s1):1 (n3,s3):3] next=13
I170925 20:35:45.365499 61462 storage/store.go:3237  [s2,r1/12:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:45.372518 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 54686bdd at index 67
I170925 20:35:45.379670 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 106, log entries: 57, rate-limit: 2.0 MiB/sec, 6ms
I170925 20:35:45.384755 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):13): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=13]
I170925 20:35:45.394584 61842 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):13): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):13] next=14
I170925 20:35:45.406707 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):13): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):13, next=14]
I170925 20:35:45.438756 61859 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):13): updated=[(n1,s1):1 (n3,s3):3] next=14
I170925 20:35:45.450646 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 3e085b2f at index 71
I170925 20:35:45.460292 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 112, log entries: 61, rate-limit: 2.0 MiB/sec, 9ms
I170925 20:35:45.462241 61877 storage/replica_raftstorage.go:729  [s2,r1/13:/M{in-ax}] applying preemptive snapshot at index 71 (id=3e085b2f, encoded size=46662, 1 rocksdb batches, 61 log entries)
I170925 20:35:45.471742 61877 storage/replica_raftstorage.go:735  [s2,r1/13:/M{in-ax}] applied preemptive snapshot in 9ms [clear=0ms batch=0ms entries=7ms commit=1ms]
I170925 20:35:45.472889 61298 storage/store.go:2168  [replicaGC,s2,r1/13:/M{in-ax}] removing replica
I170925 20:35:45.474093 61298 storage/replica.go:762  [replicaGC,s2,r1/13:/M{in-ax}] removed 34 (26+8) keys in 1ms [clear=0ms commit=0ms]
W170925 20:35:45.476197 61459 storage/store.go:3244  [s1] got error from r1, replica (n2,s2):13: raft group deleted
I170925 20:35:45.480041 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):14): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=14]
W170925 20:35:45.484405 61459 storage/store.go:3244  [s1] got error from r1, replica (n2,s2):13: raft group deleted
I170925 20:35:45.492686 61892 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):14): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):14] next=15
I170925 20:35:45.528355 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):14): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):14, next=15]
I170925 20:35:45.545241 61787 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):14): updated=[(n1,s1):1 (n3,s3):3] next=15
I170925 20:35:45.548257 61087 storage/replica_raftstorage.go:523  [raftsnapshot,s1,r1/1:/M{in-ax}] generated Raft snapshot 690bf10f at index 75
I170925 20:35:45.556265 61087 storage/store.go:3503  [raftsnapshot,s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):14: kv pairs: 119, log entries: 65, rate-limit: 8.0 MiB/sec, 7ms
I170925 20:35:45.560819 61662 storage/replica_raftstorage.go:729  [s2,r1/14:{-}] applying Raft snapshot at index 75 (id=690bf10f, encoded size=49716, 1 rocksdb batches, 65 log entries)
I170925 20:35:45.567601 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 7f1dfeeb at index 76
I170925 20:35:45.581589 61662 storage/replica_raftstorage.go:735  [s2,r1/14:/M{in-ax}] applied Raft snapshot in 21ms [clear=0ms batch=0ms entries=19ms commit=1ms]
I170925 20:35:45.584082 61462 storage/store.go:3237  [s2,r1/14:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:45.590093 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 119, log entries: 66, rate-limit: 2.0 MiB/sec, 6ms
I170925 20:35:45.591624 61664 storage/replica_raftstorage.go:729  [s2,r1/14:/M{in-ax}] applying preemptive snapshot at index 76 (id=7f1dfeeb, encoded size=50334, 1 rocksdb batches, 66 log entries)
I170925 20:35:45.598905 61664 storage/replica_raftstorage.go:735  [s2,r1/14:/M{in-ax}] applied preemptive snapshot in 7ms [clear=0ms batch=0ms entries=5ms commit=1ms]
I170925 20:35:45.599576 61298 storage/store.go:2168  [replicaGC,s2,r1/14:/M{in-ax}] removing replica
I170925 20:35:45.604896 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):15): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=15]
I170925 20:35:45.608430 61298 storage/replica.go:762  [replicaGC,s2,r1/14:/M{in-ax}] removed 34 (26+8) keys in 1ms [clear=0ms commit=1ms]
I170925 20:35:45.649191 61913 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):15): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):15] next=16
I170925 20:35:45.660658 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):15): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):15, next=16]
I170925 20:35:45.675200 61900 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):15): updated=[(n1,s1):1 (n3,s3):3] next=16
I170925 20:35:45.683518 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 61db3ff6 at index 82
I170925 20:35:45.693043 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 127, log entries: 72, rate-limit: 2.0 MiB/sec, 9ms
E170925 20:35:45.693650 61462 storage/store.go:3235  [s2,r1/15:{-}] unable to add to replica GC queue: replica not initialized
I170925 20:35:45.695698 61866 storage/replica_raftstorage.go:729  [s2,r1/15:{-}] applying preemptive snapshot at index 82 (id=61db3ff6, encoded size=54495, 1 rocksdb batches, 72 log entries)
I170925 20:35:45.708249 61866 storage/replica_raftstorage.go:735  [s2,r1/15:/M{in-ax}] applied preemptive snapshot in 12ms [clear=0ms batch=0ms entries=9ms commit=1ms]
I170925 20:35:45.716746 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):16): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=16]
I170925 20:35:45.728004 61792 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):16): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):16] next=17
I170925 20:35:45.747583 61462 storage/store.go:3237  [s2,r1/15:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:45.772954 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):16): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):16, next=17]
I170925 20:35:45.804952 61971 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):16): updated=[(n1,s1):1 (n3,s3):3] next=17
I170925 20:35:45.821485 61462 storage/store.go:3237  [s2,r1/16:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:45.826216 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 70f30e5e at index 86
I170925 20:35:45.836941 61462 storage/store.go:3237  [s2,r1/16:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:45.845275 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 133, log entries: 76, rate-limit: 2.0 MiB/sec, 18ms
I170925 20:35:45.849603 61298 storage/store.go:2168  [replicaGC,s2,r1/16:/M{in-ax}] removing replica
I170925 20:35:45.851062 61298 storage/replica.go:762  [replicaGC,s2,r1/16:/M{in-ax}] removed 34 (26+8) keys in 1ms [clear=0ms commit=0ms]
I170925 20:35:45.854588 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):17): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=17]
I170925 20:35:45.880083 61990 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):17): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):17] next=18
I170925 20:35:45.896109 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):17): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):17, next=18]
I170925 20:35:45.908329 61943 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):17): updated=[(n1,s1):1 (n3,s3):3] next=18
I170925 20:35:45.927113 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 2f429629 at index 90
E170925 20:35:45.935613 61462 storage/store.go:3235  [s2,r1/17:{-}] unable to add to replica GC queue: replica not initialized
I170925 20:35:45.943872 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 139, log entries: 80, rate-limit: 2.0 MiB/sec, 16ms
I170925 20:35:45.955644 61868 storage/replica_raftstorage.go:729  [s2,r1/17:{-}] applying preemptive snapshot at index 90 (id=2f429629, encoded size=60863, 1 rocksdb batches, 80 log entries)
I170925 20:35:45.984236 61868 storage/replica_raftstorage.go:735  [s2,r1/17:/M{in-ax}] applied preemptive snapshot in 28ms [clear=0ms batch=0ms entries=26ms commit=1ms]
I170925 20:35:46.038832 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):18): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=18]
I170925 20:35:46.067066 61992 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):18): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):18] next=19
I170925 20:35:46.088565 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):18): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):18, next=19]
I170925 20:35:46.115315 61945 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):18): updated=[(n1,s1):1 (n3,s3):3] next=19
I170925 20:35:46.151142 61462 storage/store.go:3237  [s2,r1/18:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:46.152823 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 3cd10ac5 at index 97
I170925 20:35:46.155742 61298 storage/store.go:2168  [replicaGC,s2,r1/18:/M{in-ax}] removing replica
I170925 20:35:46.157794 61298 storage/replica.go:762  [replicaGC,s2,r1/18:/M{in-ax}] removed 34 (26+8) keys in 2ms [clear=1ms commit=1ms]
I170925 20:35:46.174642 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 148, log entries: 87, rate-limit: 2.0 MiB/sec, 20ms
I170925 20:35:46.178146 62015 storage/replica_raftstorage.go:729  [s2,r1/?:{-}] applying preemptive snapshot at index 97 (id=3cd10ac5, encoded size=65511, 1 rocksdb batches, 87 log entries)
I170925 20:35:46.194875 62015 storage/replica_raftstorage.go:735  [s2,r1/?:/M{in-ax}] applied preemptive snapshot in 16ms [clear=0ms batch=0ms entries=11ms commit=4ms]
I170925 20:35:46.199717 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):19): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=19]
I170925 20:35:46.209965 61960 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):19): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):19] next=20
I170925 20:35:46.223603 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):19): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):19, next=20]
I170925 20:35:46.239221 62030 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):19): updated=[(n1,s1):1 (n3,s3):3] next=20
I170925 20:35:46.258977 61462 storage/store.go:3237  [s2,r1/19:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:46.268000 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 4f2be70a at index 101
I170925 20:35:46.280014 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 154, log entries: 91, rate-limit: 2.0 MiB/sec, 11ms
I170925 20:35:46.282314 61298 storage/store.go:2168  [replicaGC,s2,r1/19:/M{in-ax}] removing replica
I170925 20:35:46.284675 61298 storage/replica.go:762  [replicaGC,s2,r1/19:/M{in-ax}] removed 34 (26+8) keys in 2ms [clear=1ms commit=1ms]
I170925 20:35:46.285846 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):20): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=20]
I170925 20:35:46.301227 62101 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):20): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):20] next=21
I170925 20:35:46.321404 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):20): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):20, next=21]
I170925 20:35:46.335664 61087 storage/replica_raftstorage.go:523  [raftsnapshot,s1,r1/1:/M{in-ax}] generated Raft snapshot 47bd27e5 at index 104
I170925 20:35:46.365691 61087 storage/store.go:3503  [raftsnapshot,s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):20: kv pairs: 160, log entries: 94, rate-limit: 8.0 MiB/sec, 20ms
I170925 20:35:46.367941 62031 storage/replica_raftstorage.go:729  [s2,r1/20:{-}] applying Raft snapshot at index 104 (id=47bd27e5, encoded size=71261, 1 rocksdb batches, 94 log entries)
I170925 20:35:46.368026 62108 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):20): updated=[(n1,s1):1 (n3,s3):3] next=21
I170925 20:35:46.391020 62031 storage/replica_raftstorage.go:735  [s2,r1/20:/M{in-ax}] applied Raft snapshot in 23ms [clear=1ms batch=0ms entries=15ms commit=5ms]
I170925 20:35:46.416000 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 8fcd3b3e at index 105
I170925 20:35:46.419975 61462 storage/store.go:3237  [s2,r1/20:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:46.423138 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 160, log entries: 95, rate-limit: 2.0 MiB/sec, 5ms
I170925 20:35:46.424624 62074 storage/replica_raftstorage.go:729  [s2,r1/20:/M{in-ax}] applying preemptive snapshot at index 105 (id=8fcd3b3e, encoded size=71879, 1 rocksdb batches, 95 log entries)
I170925 20:35:46.438157 61462 storage/store.go:3237  [s2,r1/20:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:46.438376 62074 storage/replica_raftstorage.go:735  [s2,r1/20:/M{in-ax}] applied preemptive snapshot in 13ms [clear=1ms batch=0ms entries=8ms commit=3ms]
I170925 20:35:46.443419 61298 storage/store.go:2168  [replicaGC,s2,r1/20:/M{in-ax}] removing replica
I170925 20:35:46.445156 61298 storage/replica.go:762  [replicaGC,s2,r1/20:/M{in-ax}] removed 34 (26+8) keys in 1ms [clear=1ms commit=1ms]
I170925 20:35:46.449791 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):21): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=21]
I170925 20:35:46.467427 62077 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):21): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):21] next=22
I170925 20:35:46.477295 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):21): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):21, next=22]
I170925 20:35:46.489305 61087 storage/replica_raftstorage.go:523  [raftsnapshot,s1,r1/1:/M{in-ax}] generated Raft snapshot 43c67b56 at index 110
I170925 20:35:46.500114 61087 storage/store.go:3503  [raftsnapshot,s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):21: kv pairs: 167, log entries: 4, rate-limit: 8.0 MiB/sec, 10ms
I170925 20:35:46.501282 62039 storage/replica_raftstorage.go:729  [s2,r1/21:{-}] applying Raft snapshot at index 110 (id=43c67b56, encoded size=13985, 1 rocksdb batches, 4 log entries)
I170925 20:35:46.508170 62039 storage/replica_raftstorage.go:735  [s2,r1/21:/M{in-ax}] applied Raft snapshot in 7ms [clear=1ms batch=0ms entries=0ms commit=0ms]
I170925 20:35:46.518757 62081 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):21): updated=[(n1,s1):1 (n3,s3):3] next=22
I170925 20:35:46.537320 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot adf4bc49 at index 112
I170925 20:35:46.542478 61462 storage/store.go:3237  [s2,r1/21:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:46.549560 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 168, log entries: 6, rate-limit: 2.0 MiB/sec, 11ms
I170925 20:35:46.550265 61462 storage/store.go:3237  [s2,r1/21:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:46.571127 61298 storage/store.go:2168  [replicaGC,s2,r1/21:/M{in-ax}] removing replica
I170925 20:35:46.572775 61298 storage/replica.go:762  [replicaGC,s2,r1/21:/M{in-ax}] removed 34 (26+8) keys in 1ms [clear=1ms commit=0ms]
I170925 20:35:46.576369 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):22): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=22]
I170925 20:35:46.587303 62138 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):22): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):22] next=23
I170925 20:35:46.632010 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):22): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):22, next=23]
I170925 20:35:46.644541 62194 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):22): updated=[(n1,s1):1 (n3,s3):3] next=23
I170925 20:35:46.675334 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 63df3d04 at index 117
E170925 20:35:46.681379 61462 storage/store.go:3235  [s2,r1/22:{-}] unable to add to replica GC queue: replica not initialized
I170925 20:35:46.682774 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 175, log entries: 11, rate-limit: 2.0 MiB/sec, 4ms
I170925 20:35:46.684557 62190 storage/replica_raftstorage.go:729  [s2,r1/22:{-}] applying preemptive snapshot at index 117 (id=63df3d04, encoded size=18763, 1 rocksdb batches, 11 log entries)
I170925 20:35:46.687401 62190 storage/replica_raftstorage.go:735  [s2,r1/22:/M{in-ax}] applied preemptive snapshot in 3ms [clear=1ms batch=0ms entries=1ms commit=0ms]
I170925 20:35:46.701311 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):23): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=23]
I170925 20:35:46.732081 62094 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):23): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):23] next=24
I170925 20:35:46.814538 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):23): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):23, next=24]
I170925 20:35:46.843956 61952 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):23): updated=[(n1,s1):1 (n3,s3):3] next=24
I170925 20:35:46.860179 61298 storage/store.go:2168  [replicaGC,s2,r1/23:/M{in-ax}] removing replica
I170925 20:35:46.873394 61462 storage/store.go:3237  [s2,r1/23:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:46.873466 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 6d484136 at index 121
I170925 20:35:46.881270 61298 storage/replica.go:762  [replicaGC,s2,r1/23:/M{in-ax}] removed 34 (26+8) keys in 20ms [clear=20ms commit=1ms]
I170925 20:35:46.882278 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot e06951a6 at index 121
I170925 20:35:46.899755 62229 storage/replica_raftstorage.go:729  [s2,r1/?:{-}] applying preemptive snapshot at index 121 (id=e06951a6, encoded size=21947, 1 rocksdb batches, 15 log entries)
I170925 20:35:46.905478 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 181, log entries: 15, rate-limit: 2.0 MiB/sec, 20ms
I170925 20:35:46.907026 62229 storage/replica_raftstorage.go:735  [s2,r1/?:/M{in-ax}] applied preemptive snapshot in 7ms [clear=1ms batch=0ms entries=5ms commit=1ms]
I170925 20:35:46.921548 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):24): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=24]
I170925 20:35:46.939140 62231 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):24): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):24] next=25
I170925 20:35:46.957219 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):24): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):24, next=25]
I170925 20:35:47.034921 62247 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):24): updated=[(n1,s1):1 (n3,s3):3] next=25
I170925 20:35:47.063379 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 08141441 at index 128
I170925 20:35:47.066501 61462 storage/store.go:3237  [s2,r1/24:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:47.068213 61298 storage/store.go:2168  [replicaGC,s2,r1/24:/M{in-ax}] removing replica
I170925 20:35:47.072696 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 1892b8a7 at index 128
I170925 20:35:47.073066 61298 storage/replica.go:762  [replicaGC,s2,r1/24:/M{in-ax}] removed 34 (26+8) keys in 4ms [clear=3ms commit=1ms]
I170925 20:35:47.080495 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 190, log entries: 22, rate-limit: 2.0 MiB/sec, 7ms
I170925 20:35:47.090377 62238 storage/replica_raftstorage.go:729  [s2,r1/?:{-}] applying preemptive snapshot at index 128 (id=1892b8a7, encoded size=26596, 1 rocksdb batches, 22 log entries)
I170925 20:35:47.096008 62238 storage/replica_raftstorage.go:735  [s2,r1/?:/M{in-ax}] applied preemptive snapshot in 5ms [clear=1ms batch=0ms entries=3ms commit=1ms]
I170925 20:35:47.100402 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):25): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=25]
I170925 20:35:47.115786 62293 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):25): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):25] next=26
I170925 20:35:47.138776 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):25): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):25, next=26]
I170925 20:35:47.170717 62299 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):25): updated=[(n1,s1):1 (n3,s3):3] next=26
I170925 20:35:47.181689 61462 storage/store.go:3237  [s2,r1/25:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:47.181745 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 0d8526ec at index 132
I170925 20:35:47.184167 61298 storage/store.go:2168  [replicaGC,s2,r1/25:/M{in-ax}] removing replica
I170925 20:35:47.185989 61298 storage/replica.go:762  [replicaGC,s2,r1/25:/M{in-ax}] removed 34 (26+8) keys in 1ms [clear=1ms commit=1ms]
I170925 20:35:47.189255 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 6d373874 at index 132
I170925 20:35:47.230454 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 196, log entries: 26, rate-limit: 2.0 MiB/sec, 40ms
I170925 20:35:47.233567 62284 storage/replica_raftstorage.go:729  [s2,r1/?:{-}] applying preemptive snapshot at index 132 (id=6d373874, encoded size=29784, 1 rocksdb batches, 26 log entries)
I170925 20:35:47.237711 62284 storage/replica_raftstorage.go:735  [s2,r1/?:/M{in-ax}] applied preemptive snapshot in 4ms [clear=1ms batch=0ms entries=2ms commit=1ms]
I170925 20:35:47.247398 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):26): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=26]
I170925 20:35:47.257713 62288 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):26): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):26] next=27
I170925 20:35:47.273398 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):26): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):26, next=27]
I170925 20:35:47.300944 62318 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):26): updated=[(n1,s1):1 (n3,s3):3] next=27
I170925 20:35:47.312209 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 7240c6e8 at index 136
I170925 20:35:47.317914 61462 storage/store.go:3237  [s2,r1/26:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:47.327867 61298 storage/store.go:2168  [replicaGC,s2,r1/26:/M{in-ax}] removing replica
I170925 20:35:47.328758 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 202, log entries: 30, rate-limit: 2.0 MiB/sec, 15ms
I170925 20:35:47.331933 61298 storage/replica.go:762  [replicaGC,s2,r1/26:/M{in-ax}] removed 34 (26+8) keys in 4ms [clear=2ms commit=2ms]
I170925 20:35:47.333589 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):27): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=27]
I170925 20:35:47.379239 62301 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):27): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):27] next=28
I170925 20:35:47.410190 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):27): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):27, next=28]
I170925 20:35:47.420837 61087 storage/replica_raftstorage.go:523  [raftsnapshot,s1,r1/1:/M{in-ax}] generated Raft snapshot 13e78a3c at index 140
I170925 20:35:47.432974 61087 storage/store.go:3503  [raftsnapshot,s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):27: kv pairs: 207, log entries: 34, rate-limit: 8.0 MiB/sec, 10ms
I170925 20:35:47.434097 62305 storage/replica_raftstorage.go:729  [s2,r1/27:{-}] applying Raft snapshot at index 140 (id=13e78a3c, encoded size=35576, 1 rocksdb batches, 34 log entries)
I170925 20:35:47.455067 62348 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):27): updated=[(n1,s1):1 (n3,s3):3] next=28
I170925 20:35:47.462406 62305 storage/replica_raftstorage.go:735  [s2,r1/27:/M{in-ax}] applied Raft snapshot in 28ms [clear=1ms batch=0ms entries=16ms commit=3ms]
I170925 20:35:47.466363 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot f502560f at index 143
I170925 20:35:47.474529 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 69c9523c at index 143
I170925 20:35:47.480394 61462 storage/store.go:3237  [s2,r1/27:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:47.487018 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 211, log entries: 37, rate-limit: 2.0 MiB/sec, 10ms
I170925 20:35:47.488524 62329 storage/replica_raftstorage.go:729  [s2,r1/27:/M{in-ax}] applying preemptive snapshot at index 143 (id=69c9523c, encoded size=37630, 1 rocksdb batches, 37 log entries)
I170925 20:35:47.494208 62329 storage/replica_raftstorage.go:735  [s2,r1/27:/M{in-ax}] applied preemptive snapshot in 5ms [clear=1ms batch=0ms entries=3ms commit=1ms]
I170925 20:35:47.494921 61298 storage/store.go:2168  [replicaGC,s2,r1/27:/M{in-ax}] removing replica
I170925 20:35:47.496608 61298 storage/replica.go:762  [replicaGC,s2,r1/27:/M{in-ax}] removed 34 (26+8) keys in 1ms [clear=1ms commit=0ms]
I170925 20:35:47.502844 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):28): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=28]
I170925 20:35:47.518115 62418 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):28): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):28] next=29
I170925 20:35:47.565207 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):28): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):28, next=29]
I170925 20:35:47.584376 61087 storage/replica_raftstorage.go:523  [raftsnapshot,s1,r1/1:/M{in-ax}] generated Raft snapshot 9d191f9d at index 147
I170925 20:35:47.586906 62268 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):28): updated=[(n1,s1):1 (n3,s3):3] next=29
I170925 20:35:47.599038 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 150ab4ae at index 148
I170925 20:35:47.623212 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 227ab4fc at index 148
I170925 20:35:47.626930 61087 storage/store.go:3503  [raftsnapshot,s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):28: kv pairs: 217, log entries: 41, rate-limit: 8.0 MiB/sec, 23ms
I170925 20:35:47.628509 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot ae03c15b at index 148
I170925 20:35:47.631716 62430 storage/replica_raftstorage.go:729  [s2,r1/28:{-}] applying Raft snapshot at index 147 (id=9d191f9d, encoded size=40220, 1 rocksdb batches, 41 log entries)
E170925 20:35:47.635374 61462 storage/store.go:3235  [s2,r1/28:{-}] unable to add to replica GC queue: replica not initialized
I170925 20:35:47.642836 62430 storage/replica_raftstorage.go:735  [s2,r1/28:/M{in-ax}] applied Raft snapshot in 6ms [clear=1ms batch=0ms entries=4ms commit=1ms]
I170925 20:35:47.660773 61462 storage/store.go:3237  [s2,r1/28:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:47.662427 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 217, log entries: 42, rate-limit: 2.0 MiB/sec, 25ms
I170925 20:35:47.690858 62372 storage/replica_raftstorage.go:729  [s2,r1/28:/M{in-ax}] applying preemptive snapshot at index 148 (id=ae03c15b, encoded size=40840, 1 rocksdb batches, 42 log entries)
I170925 20:35:47.708155 62372 storage/replica_raftstorage.go:735  [s2,r1/28:/M{in-ax}] applied preemptive snapshot in 17ms [clear=2ms batch=0ms entries=9ms commit=6ms]
I170925 20:35:47.709733 61298 storage/store.go:2168  [replicaGC,s2,r1/28:/M{in-ax}] removing replica
I170925 20:35:47.713555 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):29): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=29]
I170925 20:35:47.718987 61298 storage/replica.go:762  [replicaGC,s2,r1/28:/M{in-ax}] removed 34 (26+8) keys in 9ms [clear=1ms commit=7ms]
I170925 20:35:47.725967 62454 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):29): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):29] next=30
I170925 20:35:47.736892 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):29): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):29, next=30]
I170925 20:35:47.760285 61087 storage/replica_raftstorage.go:523  [raftsnapshot,s1,r1/1:/M{in-ax}] generated Raft snapshot 60571954 at index 151
I170925 20:35:47.778788 62394 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):29): updated=[(n1,s1):1 (n3,s3):3] next=30
E170925 20:35:47.787738 61462 storage/store.go:3235  [s2,r1/29:{-}] unable to add to replica GC queue: replica not initialized
I170925 20:35:47.790532 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 36fa2aef at index 153
I170925 20:35:47.802196 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot fa0c445a at index 153
I170925 20:35:47.802254 62392 storage/replica_raftstorage.go:729  [s2,r1/29:{-}] applying Raft snapshot at index 151 (id=60571954, encoded size=43412, 1 rocksdb batches, 45 log entries)
I170925 20:35:47.802487 61087 storage/store.go:3503  [raftsnapshot,s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):29: kv pairs: 223, log entries: 45, rate-limit: 8.0 MiB/sec, 22ms
I170925 20:35:47.808086 62392 storage/replica_raftstorage.go:735  [s2,r1/29:/M{in-ax}] applied Raft snapshot in 5ms [clear=1ms batch=0ms entries=3ms commit=1ms]
I170925 20:35:47.813077 61462 storage/store.go:3237  [s2,r1/29:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:47.816803 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 224, log entries: 47, rate-limit: 2.0 MiB/sec, 5ms
I170925 20:35:47.822216 62446 storage/replica_raftstorage.go:729  [s2,r1/29:/M{in-ax}] applying preemptive snapshot at index 153 (id=fa0c445a, encoded size=44522, 1 rocksdb batches, 47 log entries)
I170925 20:35:47.838351 62446 storage/replica_raftstorage.go:735  [s2,r1/29:/M{in-ax}] applied preemptive snapshot in 16ms [clear=1ms batch=0ms entries=3ms commit=11ms]
I170925 20:35:47.839113 61298 storage/store.go:2168  [replicaGC,s2,r1/29:/M{in-ax}] removing replica
I170925 20:35:47.845071 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):30): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=30]
I170925 20:35:47.850290 61298 storage/replica.go:762  [replicaGC,s2,r1/29:/M{in-ax}] removed 34 (26+8) keys in 11ms [clear=10ms commit=1ms]
I170925 20:35:47.857522 62460 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):30): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):30] next=31
I170925 20:35:47.875591 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):30): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):30, next=31]
I170925 20:35:47.896187 61087 storage/replica_raftstorage.go:523  [raftsnapshot,s1,r1/1:/M{in-ax}] generated Raft snapshot 8098631f at index 156
I170925 20:35:47.928096 61087 storage/store.go:3503  [raftsnapshot,s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):30: kv pairs: 228, log entries: 50, rate-limit: 8.0 MiB/sec, 31ms
I170925 20:35:47.928736 62485 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing REMOVE_REPLICA((n2,s2):30): updated=[(n1,s1):1 (n3,s3):3] next=31
I170925 20:35:47.932064 62412 storage/replica_raftstorage.go:729  [s2,r1/30:{-}] applying Raft snapshot at index 156 (id=8098631f, encoded size=46640, 1 rocksdb batches, 50 log entries)
I170925 20:35:47.939771 62412 storage/replica_raftstorage.go:735  [s2,r1/30:/M{in-ax}] applied Raft snapshot in 7ms [clear=1ms batch=0ms entries=5ms commit=1ms]
I170925 20:35:47.948413 61462 storage/store.go:3237  [s2,r1/30:/M{in-ax}] added to replica GC queue (peer suggestion)
I170925 20:35:47.967726 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot f14e8606 at index 159
I170925 20:35:47.983588 61298 storage/store.go:2168  [replicaGC,s2,r1/30:/M{in-ax}] removing replica
I170925 20:35:47.985694 61298 storage/replica.go:762  [replicaGC,s2,r1/30:/M{in-ax}] removed 34 (26+8) keys in 2ms [clear=1ms commit=1ms]
I170925 20:35:47.987650 60940 storage/replica_raftstorage.go:523  [s1,r1/1:/M{in-ax}] generated preemptive snapshot 80242c92 at index 159
I170925 20:35:47.993040 60940 storage/store.go:3503  [s1,r1/1:/M{in-ax}] streamed snapshot to (n2,s2):?: kv pairs: 232, log entries: 53, rate-limit: 2.0 MiB/sec, 5ms
I170925 20:35:47.996936 62498 storage/replica_raftstorage.go:729  [s2,r1/?:{-}] applying preemptive snapshot at index 159 (id=80242c92, encoded size=48694, 1 rocksdb batches, 53 log entries)
I170925 20:35:48.002780 62498 storage/replica_raftstorage.go:735  [s2,r1/?:/M{in-ax}] applied preemptive snapshot in 6ms [clear=1ms batch=0ms entries=3ms commit=1ms]
I170925 20:35:48.013331 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (ADD_REPLICA (n2,s2):31): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, next=31]
I170925 20:35:48.025732 62500 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing ADD_REPLICA((n2,s2):31): updated=[(n1,s1):1 (n3,s3):3 (n2,s2):31] next=32
I170925 20:35:48.036723 60940 storage/replica_command.go:3661  [s1,r1/1:/M{in-ax}] change replicas (REMOVE_REPLICA (n2,s2):31): read existing descriptor r1:/M{in-ax} [(n1,s1):1, (n3,s3):3, (n2,s2):31, next=32]
I170925 20:35:48.054329 62504 storage/replica.go:3056  [s1,r1/1:/M{in-ax}] proposing R
```
Please assign, take a look and update the issue accordingly.

--- a/pkg/storage/replica_raftstorage.go
+++ b/pkg/storage/replica_raftstorage.go
@@ -413,9 +413,12 @@ func (r *Replica) GetSnapshot(
 		defer r.raftMu.Unlock()
 		return fn(r.raftMu.sideloaded)
 	}
+	// NB: We have Replica.mu read-locked, but we need ot write-locked in order
+	// to use Replica.mu.stateLoader. This call is not performance sensitive, so
+	// create a new state loader.
 	snapData, err := snapshot(
-		ctx, r.mu.stateLoader, snapType, snap, rangeID,
-		r.store.raftEntryCache, withSideloaded, startKey,
+		ctx, makeReplicaStateLoader(r.store.cfg.Settings, rangeID), snapType,
+		snap, rangeID, r.store.raftEntryCache, withSideloaded, startKey,
 	)
 	if err != nil {
 		log.Errorf(ctx, "error generating snapshot: %s", err)

